daprdocs/content/en/reference/api/metadata_api.md

```diff
@@ -86,12 +96,14 @@ curl http://localhost:3500/v1.0/metadata
         {
             "name":"pubsub",
             "type":"pubsub.redis",
-            "version":""
+            "version":"",
```

Comment:
Every component has a version, so this cannot be blank, correct?

Reply:
Added the current version